### DISCUSSION TITLE
Welcome to ShellShockLiveAimbot Discussions!

### DISCUSSION THREAD

**Discussion by henrydatei** [henrydatei/ShellShockLiveAimbot#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cobtown5252** [comment]
hello my name is cobtown52 on steam and what happens is i was wanting to know if you will update this because this setup will have security issues and problems with usage of the mod, making it bad to use on a computer. also the mod is really old and will probably not work with the new updates to shellshock. also with the security problems it might have problems working. can you please update this for usage of education purposes?

**Reply by henrydatei** [reply]
The python script still works, I've tested it a couple days ago. I know the codebase is really old and more like a temporary solution, I might update it since my python skills have increased since I initially wrote this. 

I'm pretty sure the usage of the script will roughly stay the same, important is (and I should mention this in README.md) that you set the resolution of the game to 1920x1080, otherwise the values are not correct. 

But what do you mean with security problems?
